tools/libboard/board_n6010/board_n6010.c

```diff
@@ -257,83 +257,63 @@ fpga_result read_max10fw_version(fpga_token token, char *max10fw_ver, size_t len
 	return res;
 }
 
-// Read mac information
-fpga_result read_mac_info(fpga_token token, uint32_t afu_channel_num,
-	struct ether_addr *mac_addr)
+// print mac information
+fpga_result print_mac_info(fpga_token token)
 {
-	fpga_result res = FPGA_OK;
-	char mac_buf[MACADDR_LEN] = { 0 };
-	char mac_count[MACADDR_LEN] = { 0 };
-	uint64_t count = 0;
-	char *endptr = NULL;
-
-	if (mac_addr == NULL) {
-		OPAE_ERR("Invalid Input parameters");
-		return FPGA_INVALID_PARAM;
-	}
-
-	res = read_sysfs(token, DFL_SYSFS_MACADDR_PATH, mac_buf, MACADDR_LEN - 1);
+	fpga_result res            = FPGA_OK;
+	char buf[MAC_BUF_LEN]      = { 0 };
+	char count[MAC_BUF_LEN]    = { 0 };
+	int i                      = 0;
+	int n                      = 0;
+	char *endptr               = NULL;
+	cvl_mac mac;
+	unsigned int mac_byte[6]   = { 0 };
+
+	res = read_sysfs(token, DFL_SYSFS_MACADDR_PATH, (char *)buf, MAC_BUF_LEN - 1);
 	if (res != FPGA_OK) {
-		OPAE_ERR("Failed to get read object");
+		OPAE_ERR("Failed to read mac information");
 		return res;
 	}
 
-	ether_aton_r(mac_buf, mac_addr);
+	sscanf(buf, "%x:%x:%x:%x:%x:%x", &mac_byte[0], &mac_byte[1],
```

Comment:
ether_aton_r is a better choice than sscanf.

Reply:
fixed ether_aton_r  function.